### PR TITLE
update sleep 10s between retrievals

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

**Review by bajtos — APPROVED** [review]
:shipit:

File: lib/spark.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ export default class Spark {
         this.#activity.onError()
         console.error(err)
       }
-      await sleep(1_000)
+      await sleep(10_000)
```
Comment (bajtos) [inline lib/spark.js:138]: Should we extract this into a constant defined at the top of this file to make it easier to find?

Comment (juliangruber) [reply lib/spark.js]: good idea!

**Review by bajtos — APPROVED** [review]
👌🏻

**Comment by juliangruber** [comment]
https://github.com/filecoin-station/spark/releases/tag/v0.1.1

**Pull request merged**